README.md

```diff
@@ -1,2 +1,118 @@
-# raincatcher-workflow-angular
-An Angular 1 Implementation of the fh-wfm-workflow module
+# Raincatcher Workflow Angular User Interface
+
+This module is an Angular JS implementation of Workflows for the Raincatcher project.
+
+## Setup
+
+This module is packaged in a CommonJS format, exporting the name of the Angular namespace.  The module can be included in an angular.js as follows:
+
+```javascript
+
+var config = {
+  mode: 'admin',
```

Comment:
The mode is for each module right now, however I think in future these will be removed in favour of more configurable directives.